### DISCUSSION TITLE
Opensearch

### DISCUSSION THREAD

**Discussion by innoneo-rahulkrishnan** [mongoosastic/mongoosastic#610] [Q&A]
Hey, I am using AWS for managing my elastic search. As you may know, AWS has stopped providing elastic search updates, and instead, they are now using their own implementation called OpenSearch. Will this package support OpenSearch also?

**Comment by vprodan** [comment]
> Hey, I am using AWS for managing my elastic search. As you may know, AWS has stopped providing elastic search updates, and instead, they are now using their own implementation called OpenSearch. Will this package support OpenSearch also?

It should be fully compatible
